genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :name, :items
+
+  def initialize(name)
+    @id = rand(1000..9999)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item)
+    item.genre = self
```

Comment:
Instead of assigning `genre` directly, you should use the `#add_genre` method from the `Item` class according to the following requirement: (`item.add_genre(self)`)
- should add self as a property of the item object (by using the correct setter from the item object)

Reply:
Thanks for this feedback. You will see the changes in the next PR